docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -197,6 +197,8 @@ _Parameters_
 
 -   _state_ `Object`: Editor state.
 -   _clientId_ `string`: Block from which to find root client ID.
+-   _getDependants_ `boolean`: Apply createSelector function to call getDependants.
+-   _includeClientId_ `boolean`: Include client ID in return array.
```

Comment:
Remember to run `npm run docs:build` to update these after all the changes to selectors are ready

Reply:
I will keep that in mind

Reply:
done !